src/ralph/backends/data/base.py

```diff
@@ -33,7 +33,7 @@ class Config:
 
         extra = "forbid"
 
-    query_string: Union[str, None] = None
+    query_string: None | str = None
```

Comment:
Discussed with @jmaupetit, Ralph is still compatible with Python3.9 for now, and it would be nice to stay compatible. WDYT?

Reply:
Good point! Thanks) Updated the changes to stay compatible with python3.9) It might also be worth testing the compatibility in the CI, added a separate PR (#256) for that.